stable/cluster-backup-chart/templates/hub-backup-pod.yaml

```diff
@@ -0,0 +1,169 @@
+# Copyright Contributors to the Open Cluster Management project
+---
+apiVersion: policy.open-cluster-management.io/v1
+kind: Policy
+metadata:
+  annotations:
+    "helm.sh/hook": pre-install,post-upgrade
+    "helm.sh/hook-delete-policy": before-hook-creation
+    policy.open-cluster-management.io/categories: PR.PT Protective Technology
```

Comment:
I think below category/control is more appropriate.
```
PR.IP Information Protection Processes and Procedures
PR.IP-4: Backups of information are conducted, maintained, and tested
```

Reply:
thanks !
Updated to 
```
    policy.open-cluster-management.io/categories: PR.IP Information Protection Processes and Procedures
    policy.open-cluster-management.io/controls: PR.IP-4 Backups of information are conducted, maintained, and tested
    policy.open-cluster-management.io/standards: NIST-CSF

```